documentation/asciidoc/microcontrollers/debug-probe/swd-connection.adoc

```diff
@@ -1,10 +1,12 @@
-== Serial Wire Debug (SWD)
+== Starting a Debug Session
 
-Serial Wire Debug (SWD) is a two-pin interface (https://developer.arm.com/documentation/101761/1-0/Debug-and-trace-interface/Serial-Wire-Debug-signals[SWDIO and SWCLK]) alternative to the JTAG four- or five-pin debugging interface standard.
+The Debug Probe will let you load binaries via the SWD port and OpenOCD: you will not need to unplug, and then push-and-hold, the BOOTSEL button every time you push a new binary to your Pico. Using the Debug Probe to upload new binaries is an entirely hands-off affair.
```

Comment:
nitpick: IMHO "then push-and-hold, the BOOTSEL button every" ought to be changed to "then push-and-hold the BOOTSEL button, every" (because you're not unplugging the BOOTSEL button :wink: ).

Reply:
This copy-pastes the prior bad grammar.

Reply:
Ahh, I hadn't noticed that. Will leave this for future copy-editors then.